Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = props => {
+    return (
+        <div className="header-container-title">
+            <div className="header-user-name">Lambda School </div>
+            <div className ="header-user-username"> @LambdaSchool</div>
+            <div className="header-space">&bull;</div>
+            <span className="header-user-date"> 26 Jan</span>
```

Comment:
Your class naming convention is interesting. Have you researched BEM at all? You might like it.